opentelemetry-aws/src/lib.rs

```diff
@@ -86,6 +86,107 @@ pub mod trace {
         _private: (),
     }
 
+    pub fn deserialize_span_context(value: &str) -> Result<SpanContext, ()> {
+        let parts: Vec<(&str, &str)> = value
+            .split_terminator(';')
+            .filter_map(from_key_value_pair)
+            .collect();
+
+        let mut trace_id: TraceId = TraceId::INVALID;
```

Comment:
It's usually best to avoid type annotations unless they are actually necessary, which doesn't seem to be the case in this block of bindings.

Reply:
Removed unnecessary type annotations